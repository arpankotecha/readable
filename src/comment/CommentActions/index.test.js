import * as actions from './index'

it('creates a new addComment action', () => {
  const addComment = actions.addComment({"id": 1})
  expect(addComment).toMatchSnapshot()
})

it('creates a new deleteComment action', () => {
  const deleteComment = actions.deleteComment({"id": 1})
  expect(deleteComment).toMatchSnapshot()
})

it('creates a new updateCommentVoteCount action', () => {
  const updateCommentVoteCount = actions.updateCommentVoteCount({"id":1}, 2)
  expect(updateCommentVoteCount).toMatchSnapshot()
})

it('creates a new incrementCommentCount action', () => {
  const incrementCommentCount = actions.incrementCommentCount(1)
  expect(incrementCommentCount).toMatchSnapshot()
})

