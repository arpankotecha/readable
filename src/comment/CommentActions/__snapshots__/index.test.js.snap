// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a new addComment action 1`] = `
Object {
  "comment": Object {
    "id": 1,
  },
  "type": "ADD_COMMENT",
}
`;

exports[`creates a new deleteComment action 1`] = `
Object {
  "comment": Object {
    "id": 1,
  },
  "type": "DELETE_COMMENT",
}
`;

exports[`creates a new incrementCommentCount action 1`] = `
Object {
  "postId": 1,
  "type": "INCREMENT_COMMENT_COUNT",
}
`;

exports[`creates a new updateCommentVoteCount action 1`] = `
Object {
  "comment": Object {
    "id": 1,
  },
  "type": "UPDATE_COMMENT_VOTE_COUNT",
  "voteCount": 2,
}
`;
